packages/notifi-dapp-example/src/components/HistoryList.tsx

```diff
@@ -71,7 +71,7 @@ export const HistoryList: React.FC<HistoryListProps> = ({
           />
         </div>
       ) : null}
-      <div className="min-h-0 overflow-scroll grow">
+      <div className="min-h-0 grow">
```

Comment:
This will also make "inbox" row scrollable. Not sure if it is a expected behavior?

Reply:
Ah, you are right,  have updated this to overflow-y-scroll as Alex and Kai has the horizontal line in their alert history view, which I can't reproduce